### PR TITLE
fs: fix scrolling by being honest about fixedHeight

### PR DESCRIPTION
**Pull request by songgao**
also cleanup the divider stuff, and placeholder icon fix.

### CODE REVIEW

File: shared/fs/index.js [hunk 1]
```diff
@@ -19,30 +19,34 @@ type FolderProps = {
   routePath: I.List<string>,
 }
 
+export const wrapRow = (rowContent, key: string) => (
```
Comment (chrisnojima) [inline shared/fs/index.js]: don't make helper functions that return a component, just make it a component
```
const WrapRow = ({children, id}) => 
```

File: shared/fs/index.js [hunk 2]
```diff
@@ -19,30 +19,34 @@ type FolderProps = {
...
+    <Divider key="divider" style={stylesDivider} />
+  </Box>
+)
+
 class Files extends React.PureComponent<FolderProps> {
   _rowRenderer = (index: number, item: Types.RowItem) => {
     switch (item.rowType) {
       case 'placeholder':
-        return <Placeholder key={`placeholder:${item.name}`} />
+        return wrapRow(<Placeholder type={item.type} />, `placeholder:${item.name}`)
```
Comment (chrisnojima) [inline shared/fs/index.js]: then
```
<WrapRow id=``><Placeholder ...
```

File: shared/fs/index.js (hunk 1 above)
Comment (songgao) [reply shared/fs/index.js]: Sure; Does this have something to do with re-rendering?

Comment (chrisnojima) [reply shared/fs/index.js]: no really its just if its a function that takes in a component and returns a component, then it itself is a component. we don't need 2 forms of this

Comment (songgao) [reply shared/fs/index.js]: Got it thanks!

**Comment by songgao** [comment]
@chrisnojima This is ready for another look; thanks!

**Pull request merged**